src/core.js

```diff
@@ -478,13 +478,7 @@ function extractStacktrace( e, offset ) {
 
 	var stack, include, i;
 
-	if ( e.stacktrace ) {
-
-		// Opera 12.x
-		return e.stacktrace.split( "\n" )[ offset + 3 ];
-	} else if ( e.stack ) {
-
-		// Firefox, Chrome, Safari 6+, IE10+, PhantomJS and Node
```

Comment:
@scottgonzalez: here we basically support everything but IE6 to 9, support for Safari 6 and 7 is at the next branch.

I wasn't able to identify how to mention it "Doesn't support IE6 to IE9", as line [477](https://github.com/jquery/qunit/pull/773/files#diff-7eb52b366866677666470e019283c8eaR477)